lib/watcher.js

```diff
@@ -123,6 +123,7 @@ class Watcher {
 				.then(runStatus => {
 					runStatus.previousFailCount = this.sumPreviousFailures(currentVector);
 					logger.finish(runStatus);
+					logger.write('[ava] To rerun tests, enter `r`\n[ava] To update snapshots used in the previous test run, enter `u`');
```

Comment:
I don't think this needs to be prefixed with `[ava]`. Currently the key needs to be followed by `Enter` in order to be recognized. How about:

> To rerun all tests, type 'r', followed by Enter
> To update snapshots used in the previous tests, type 'u', followed by Enter

Perhaps using `chalk.gray.dim` as the color.